doc/nrf/protocols/matter/getting_started/tools.rst

```diff
@@ -184,6 +204,56 @@ You can also install the ZAP tool using one of the following methods:
 
 Both these methods still require adding the ZAP tool location to the system :envvar:`PATH` environment variables, as detailed in the step 3 above.
 
+.. _ug_matter_gs_tools_cert:
+
+CHIP Certificate Tool
+*********************
+
+Matter's CHIP Certificate Tool, in short chip-cert, is a command-line utility tool for generating and editing Matter certificates such as Certificate Declarations (CD), Device Attestation Certificates (DAC), Product Attestation Intermediate (PAI) certificate, alongside their related keys.
+You can use it for integration testing purposes while working on a :ref:`Matter end product <ug_matter_intro_device>`.
+
+For more information about the chip-cert tool, see how to :ref:`generate custom certification declarations <ug_matter_device_configuring_cd_generating_steps>` for integration testing in the |NCS|.
+You can also take a look at the `CHIP Certificate Tool source files`_.
+
+.. _ug_matter_gs_tools_cert_installation:
+
+Installing CHIP Certificate Tool
+================================
+
+To install the chip-cert tool, complete the following steps:
+
+1. Navigate to the :file:`connectedhomeip` root directory.
+#. In a terminal, run the following command to build the tool executable file:
+
+   .. code-block:: console
+
+      cd src/tools/chip-cert && gn gen out && ninja -C out chip-cert
+
+#. Add the chip-cert tool to the system :envvar:`PATH` environment variable when built.
+
+Generating custom certificates in factory data
+==============================================
+
+Adding the chip-cert tool to to the system :envvar:`PATH` allows you to build :ref:`matter_samples` and the :ref:`Matter weather station <matter_weather_station_app>` application with custom certificates included in the factory data.
+This lets you for example change the test Vendor ID, Product ID, or other data.
+
+To build a Matter application in the |NCS| with custom certification data, make sure to set the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_USE_DEFAULT_CERTS` to ``n`` when :doc:`building an example with factory data <matter:nrfconnect_factory_data_configuration>`.
+
+.. _ug_matter_gs_tools_spake2:
+
+SPAKE2+ Python tool
+*******************
+
+SPAKE2+ Python Tool is a Python script for generating SPAKE2+ protocol parameters.
+The protocol is used during Matter commissioning to :ref:`establish a secure session <ug_matter_overview_commissioning_stages_case>` between the commissioner and the commissionee.
+
+.. note::
+   Currently, the tool only supports generating Verifier parameters.
+
+For usage examples, see the `SPAKE2+ Python Tool page`_ in the Matter SDK official documentation.
```

Comment:
@Damian-Nordic , could you have a look at these Spake2+ lines?

Reply:
LGTM